staff/jordi-pujol/cv/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Jordi_cv</title>
+</head>
+<body>
+    <header>
+        <h1>Jordi Pujol</h1>
+        <img src="https://img.elcomercio.pe/files/article_content_ec_fotos/uploads/2018/10/06/5bb8ce60353c5.jpeg" alt="">
+        <ul>
+            <li><h2>Web Designer Director</h2></li>
+            <li><h2>E-mail:ddddddd@gmail.com</h2></li>
+            <li><h2>Telephone: 83723984729</h2></li>
+        </ul>
+    </header>
+
+    <main>
+        <br>
+        <section><h2>Profile</h2>
+            <ul>
```

Comment:
ul and li for just listing ONE item has NO sense. please, remove unnecessary tags.